tests/test_functions.py

```
@@ -156,4 +158,16 @@ def test_functions(name, types):
 @pytest.mark.extra
 def test_extra_functions():
     validate_function("string.matchRegEx", [Type.STRING | Type.NULL, Type.STRING | Type.NULL])
-    validate_function("string.replaceRegEx", [Type.STRING | Type.NULL, Type.STRING | Type.NULL, Type.STRING | Type.NULL])
\ No newline at end of file
+    validate_function("string.replaceRegEx", [Type.STRING | Type.NULL, Type.STRING | Type.NULL, Type.STRING | Type.NULL])
+
+def query_exception(query: str, message: str, params=None):
+    try:
+        common.g.query(query, params)
+        assert False, "Expected an error"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)